settings.gradle.kts

```diff
@@ -33,3 +33,9 @@ include(":testng-test-osgi")
 
 enableFeaturePreview("VERSION_CATALOGS")
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
+
+for (project in rootProject.children) {
+    project.apply {
+        buildFileName = "${project.name}-build.gradle.kts"
```

Comment:
Not sure it is a good idea to override the default convention.
What is the need?

Reply:
This is a recommended approach in many places I have seen. The idea is that you don't confuse yourself with multiple build files with the same name. Gradle will let you name your build file to the module name as well and still honour the build file. 

I don't see any problem with that.

Reply:
ok